packages/api/test/mocks/db/post_graphql.js

```diff
@@ -63,6 +63,10 @@ module.exports = ({ body }) => {
     return gqlOkResponse(require('../../fixtures/find-content-by-cid.json'))
   }
 
+  if (body.query.includes('findMetrics')) {
+    return gqlOkResponse(require('../../fixtures/find-metrics.json'))
```

Comment:
yeah, this is the part that bit me... my fixture was a copy/paste of the graphql response, and our `gqlOkResponse` adds the `data` prop for you, but then the db client removes that prop from the response that it returns to your code. I think i'd prefer the redundant `.data` prop to just appear everywhere, so fixtures match what you get from the db, and you can look at the fixture and write your api code against it.

Reply:
yerp, it's a different `data` prop for a GQL response https://graphql.org/learn/serving-over-http/#response - it _has_ to be added in at this level.